rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml

```diff
@@ -0,0 +1,21 @@
+title: PE File Execution via Vsjitdebugger
+id: 4b51f73f-1583-4202-a8e0-2d4bbf4beeee
+status: experimental
+description: Detects the execution of Vsjitdebugger tool as parent process which is utilized like proxy for other PE files executions.
+references:
+    - https://lolbas-project.github.io/lolbas/OtherMSBinaries/Vsjitdebugger/
+author: 'Ensar Şamil, @sblmsrsn, OSCD Community' 
```

Comment:
Hello @esebese! Could you please change `OSCD Community` to the `oscd.community` or `@oscd_initiative` just like you did for the other rules? thank you!

Reply:
Thanks for mentioning this detail.

@esebese - just a `ping`, when you have time, please make the changes.

Reply:
I edited.

Reply:
Thanks!